electron/index.js

```diff
@@ -128,3 +140,5 @@ app.on('activate', () => {
     createWindow();
   }
 });
+
+app.setPath('userData', path.join(app.getPath('appData'), 'Parity-UI'));
```

Comment:
Why is this needed?

Reply:
Otherwise `userData` is `~/.config/Electron/`. Comment from the commit:

```
Electron stores the user data by default in `~/.config/<app name>`

The app name is retrieved from the package.json when Electron is
run on a folder (e.g. `electron .`); default app name 'Electron'
is used if Electron is run directly on a script, as is the case.

Electron can be run either on .build/ or on .electron/ (dev),
so it's not practical to use package.json to set the app name
(.electron/ has no package.json); hence it's better in our case
to define it in the code.
```

Setting the app name programatically doesn't seem to have an effect on the default `userData` value (it's probably computed only once at the beginning).

Reply:
userData will be `~/.config/Electron/` when running electron, but will be `~/.config/parity-ui/` once builded.

I think we can keep `app.getPath('userData')` everywhere?

Reply:
Deleted my previous comment because I just understood your reasoning.

A small remark: should it be `parity-ui` lowercase? When I run the binaries on mac, they have `./path/to/parity-ui` as their userData path.

If it's indeed lowercase, maybe use `import { name } from '../package.json'` instead of hardcoding a string.

Reply:
Okay, let's switch to lowercase then! Didn't think of importing from package.json, good idea!